framework/Source/Operations/Utils.h

```diff
@@ -0,0 +1,10 @@
+#include <metal_stdlib>
+using namespace metal;
+
+#ifndef Utils_h
+#define Utils_h
+
+float rand(float2 co);
```

Comment:
Not really sure about what this function does. 
Does it take two values and generate a random number?

Reply:
I don't know, but there are 2 metal files that had it, so I moved it to an additional file since the compiler will compile all shader files into 1, and the duplicated name caused a compile error while building the demo app